torch/package/package_exporter.py

```diff
@@ -23,7 +23,7 @@
 )
 
 import torch
```

Comment:
shouldn't we actually be removing torch lol

Reply:
Indeed haha